### ISSUE TITLE
get permaLink of a message [help needed]

### ISSUE THREAD

**Issue by vyk2rr** [howdyai/botkit#1383]
TLDR; Basically I just want to get the permalink of a message just posted but the `api.bot.chat.getPermalink` does not exist into `bot.api.chat`, Question: Do I have to do something special to get this available? or... 

[here](https://botkit.ai/docs/readme-slack.html#using-the-slack-web-api) it says that `bot.api...` would let me get access to any of these methods https://api.slack.com/methods and, I'm trying to use this https://api.slack.com/methods/chat.getPermalink by `bot.api.chat.getPermalink({` but this is what i'm getting available under bot.api.chat 
![image](https://user-images.githubusercontent.com/189216/41315711-ef6808e2-6e55-11e8-8a99-2c15d641683e.png)

this is what I have : 

```js
            bot.say({
              "text": `something here added by <@${user_data.user.id}>`,
              "channel": "channel-for-updates"
            }, (err, response) => {
  
             // then, currently this is not working obviously (getPermalink is not existing here)
              bot.api.chat.getPermalink({
                channel: "channel-for-updates",
                message_ts: response.ts
              }, (err, response) => {

               // then update an interactive message with the permalink
                bot.replyInteractive(user_data.message, {
                  "text": `I added a confirmation message <permalinkHere!>`,
                  "callback_id": 'type_of_matching',
                  "attachment_type": 'default'
                });

                bot.dialogOk();    
              });
            });
```

and finally, here there are the scopes:
![image](https://user-images.githubusercontent.com/189216/41358192-7fa064bc-6eed-11e8-9379-6f01637bf591.png)

**Comment by peterswimm** [comment]
This is probably a question better asked of Slack support directly.
